tests/core/test_asr_metrics.py

```diff
@@ -119,8 +118,19 @@ def get_wer(prediction: str, reference: str):
             n2 = random.randint(0, 512)
             s1 = __randomString(n1)
             s2 = __randomString(n2)
-            logging.debug(n1)
-            logging.debug(s1)
-            logging.debug(n2)
-            logging.debug(s2)
```

Comment:
Please leave these debug statements for @okuchaiev to review, they were added by @blisc

Reply:
Logging shouldn't be part of any test.

Reply:
This test is randomly and sometimes randomly fails. That’s why the logging is there so we can catch the corner cases where it fails

Reply:
Please keep these logging messages. Yes I understand logging should not be a part of any **working test**. The caveat here is that this test is not working as you have mentioned in your comments. So please keep these so it can log s1 and s2 in the cases where the tests fail. That way we can debug the root causes of these failures.

Reply:
yes, test fails occasionally - I need that info when it fails to diagnose.